### DISCUSSION TITLE
【羊村】组织成员榜单

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/home#244] [Announcements]
欢迎各位贡献者参与留言：

```
> 更新时间: 2023-08-09

- [别名](github地址):  [项目名称 - 角色](https://github.com/apachecn/home)

角色: 项目贡献者(贡献/宣传/翻译)、项目负责人/组织运维民工【如果都没，可以填写：组织摸鱼民工(摸鱼/装逼/吃瓜/开车/种草)】
```

例如：

> 更新时间: 2023-08-09

- [片刻小哥哥](https://github.com/jiangzhonglian): [Home - 组织运维民工](https://github.com/apachecn/.github)

---

- 组织未来赚到钱了，会联系大家，给各位发纪念品（有个念想，万一成了呢）

**Comment by jiangzhonglian** [comment]
> 更新时间: 2023-08-09

- [片刻小哥哥](https://github.com/jiangzhonglian): [Home - 组织运维民工](https://github.com/apachecn/.github)

**Comment by 86236291** [comment]
> 更新时间: 2023-08-09

- [是,但求其爱](https://github.com/86236291): [None - 组织摸鱼民工]()

**Comment by runzhi214** [comment]
> 更新时间: 2023-08-09
- [runzhi](https://github.com/runzhi214): [Cython文档 - 项目翻译者](https://github.com/apachecn/cython-doc-zh) 
- [runzhi](https://github.com/runzhi214): [Pytorch文档 - 项目贡献者](https://github.com/apachecn/pytorch-doc-zh)

**Comment by mikechengwei** [comment]
> 更新时间: 2023-08-09

- [mikechengwei](https://github.com/mikechengwei):  [kafka - 组织摸鱼民工](https://github.com/apachecn/kafka-doc-zh)

**Comment by stone28615** [comment]
[拂石](https://github.com/stone28615):  [None - 组织摸鱼民工](https://github.com/orgs/apachecn/discussions/244)

**Comment by Fadegentle** [comment]
- [伊言](https://github.com/Fadegentle): [Pytorch文档 - 项目翻译者](https://github.com/apachecn/pytorch-doc-zh)

**Comment by nicetomeet-you** [comment]
[αβ](https://github.com/nicetomeet-you)：[小红薯宣传-组织种草民工]( http://xhslink.com/XDgket)

**Comment by tenghf** [comment]
[Timor](https://github.com/tenghf) :[None-组织摸鱼民工](https://github.com/orgs/apachecn/discussions/244)

**Comment by shiyuekitty** [comment]
None - 组织摸鱼民工

**Comment by masteryi-0018** [comment]
> 更新时间: 2023-08-11

- [masteryi-0018](https://github.com/masteryi-0018):  [Pytorch文档 - 项目贡献者](https://github.com/apachecn/pytorch-doc-zh)

**Comment by ad153** [comment]
组织摸鱼民工

**Comment by AllenWrong** [comment]
> 更新时间: 2023-08-09

[Andrew Guan](https://github.com/AllenWrong):  [None - 组织摸鱼民工](https://github.com/AllenWrong)

**Comment by moye12325** [comment]
[莫叶](https://github.com/moye12325):  [None - 组织摸鱼民工](https://github.com/moye12325)

**Comment by sci-m-wang** [comment]
[云端听茗](https://github.com/sci-m-wang): [Pytorch文档 - 项目翻译者](https://github.com/apachecn/pytorch-doc-zh)

**Comment by Moon0015** [comment]
> 更新时间: 2023-12-05

- [月_moon_](https://github.com/Chengbingyi):  [Pytorch文档 - 项目翻译者](https://github.com/apachecn/pytorch-doc-zh)

**Comment by jiangzhonglian** [comment]
1
